### PR TITLE
Convert readme from Markdown to reStructuredText for pypi.

### LINKED ISSUE #9: Convert README.me to README.rst
pypi uses README.rst

### PR DESCRIPTION
**Pull request by AceFire6**
Fixes #9

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+             }
+        }
+    }
+
+Testing
+-------
+
+Install development requirements:
+
+::
```
Comment (stefanor) [inline README.rst]: You can end the previous paragraph with `::` to achieve the same thing.

Comment (AceFire6) [inline README.rst]: Truth be told, I didn't convert this by hand.

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+Tests can then be run by doing::
+
+    nosetests
+
+The integration tests require a host API key. They can be run with::
+
+    nosetests tests/test_integration.py
+
+.. |Build Status| image:: https://travis-ci.org/yola/pycloudflare.svg
```
Comment (adrianmoisey) [inline README.rst]: While you're here, can you make the image point to master?
`https://travis-ci.org/yola/pycloudflare.svg?branch=master`

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by adrianmoisey** [comment]
:+1:

Can you hold off on merging this? I'd like to test an upload job going for this and https://github.com/yola/pycloudflare/pull/18

(see https://github.com/yola/yolatools/pull/383)

**Pull request merged**